docker/README.md

```diff
@@ -27,14 +27,15 @@ sudo systemctl restart docker
 In the `docker` directory, run the following command to build the Docker image and tag it as `seismic-deeplearning`: 
 
 ```bash
-sudo docker build -t seismic-deeplearning . 
+sudo docker build -t seismic-deeplearning .
 ```
 This process will take a few minutes to complete. 
 
 # Run the Docker image:
-Once the Docker image is built, you can run it anytime using the following command: 
+Once the Docker image is built, you can run it anytime using the following command:
 ```bash
-sudo docker run --rm -it -p 8888:8888 --gpus=all --shm-size 11G --mount type=bind,source=$PWD/hrnetv2_w48_imagenet_pretrained.pth,target=/home/username/models/hrnetv2_w48_imagenet_pretrained.pth seismic-deeplearning
+sudo nvidia-docker run --rm -it -p 9000:9000 -p 9001:9001 --shm-size 11G --mount type=bind,source=$PWD/hrnetv2_w48_imagenet_pretrained.pth,target=/home/username/models/hrnetv2_w48_imagenet_pretrained.pth seismic-deeplearning
```

Comment:
This doesn't follow nvidia-docker's usage pattern (see here: https://github.com/NVIDIA/nvidia-docker#usage) 

Please also see this note from their readme: "Note that with the release of Docker 19.03, usage of nvidia-docker2 packages are deprecated since NVIDIA GPUs are now natively supported as devices in the Docker runtime." 

I think I might have used a newer version of docker  than you, thats why it worked for me probably and not for you?

Reply:
We might need to ask the user to update the Docker version 19.03 or higher like George suggested